genomic_data_service/search.py

```diff
@@ -136,3 +142,32 @@ def region_search():
         'regions': region_service.regions,
         'regions_per_file': region_service.regions_per_file
     })
+
+def get_sequence(assembly, coordinate, window=50):
+    if assembly == 'GRCh38':
+        seq_reader = py2bit.open(TWO_BIT_HG38_FILE_PATH)
+    else:
+        seq_reader = py2bit.open(TWO_BIT_HG19_FILE_PATH)
```

Comment:
```suggestion
    if assembly == 'GRCh38':
        seq_reader = py2bit.open(TWO_BIT_HG38_FILE_PATH)
    elif assembly == 'hg19':
        seq_reader = py2bit.open(TWO_BIT_HG19_FILE_PATH)
    else:
        raise ValueError("Assembly has to be either GRCh38 or hg19.")
```

Reply:
probably safer? could cause headache if someone tries some synonym (like using "hg38" and unexpectedly and silently getting hg19 sequence back).

Reply:
assembly is get from function extract_search_params(), which will only return valid assembly. Only issue right now is valid assembly includes mouse assembly which we won't use(see constant GENOME_TO_ALIAS). So we may want to remove mouse assembly at some time point.

Reply:
there is a 3rd human assembly (CHM13/T2T) that we may move to in the future.